EssentialCSharp.Web/Services/IReferralService.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Security.Claims;
+
+namespace EssentialCSharp.Web.Services;
```

Comment:
Thoughts on moving this to a "Referral" namespace rather than using a catch-all "Services" namespace.